tests/forms_tests/field_tests/test_filefield.py

```diff
@@ -80,3 +80,32 @@ def test_disabled_has_changed(self):
 
     def test_file_picklable(self):
         self.assertIsInstance(pickle.loads(pickle.dumps(FileField())), FileField)
+
+    def test_multiple(self):
+        f = FileField(multiple=True)
+        files = [SimpleUploadedFile('file_%s' % i, ('%s' % i).encode('utf-8')) for i in range(2)]
+        self.assertEqual(files, f.clean(files))
+
+        # ensure validation is executed for all files
+        f = FileField(multiple=True)
+        files = [SimpleUploadedFile('file_%s' % i, b'') for i in range(2)]
+        with self.assertRaisesMessage(ValidationError, 'The submitted file is empty.'):
```

Comment:
Would be handy if this tells which was was empty. Or change the message to - "One of the submitted files was empty".

Reply:
That is a good point. It will make the implementation more complex tho. What do you think? Worth it or not?

Reply:
Validation is currently done per file base. That is something that I don't want to change. I will add the file name to the message to be more verbose. Caveat: There could be multiple empty files. Since we raise `ValidationErrors` only the first one will be shown. That is the same for all Validators tho. I don't think this is really unexpected.